src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/dto/PlanCsvRecord.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.supportadditionalneedsapi.dto
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
+import uk.gov.justice.digital.hmpps.supportadditionalneedsapi.config.csv.CsvSerializable
+import uk.gov.justice.digital.hmpps.supportadditionalneedsapi.domain.entity.PlanCreationScheduleEntity
+
+@CsvSerializable
+@JsonPropertyOrder("reference", "prison_number", "created_at_prison", "deadline_date", "status")
+data class PlanCsvRecord(
+  @JsonProperty("reference")
+  val reference: String,
+
+  @JsonProperty("prison_number")
```

Comment:
If we are snake casing here ..... do we also need the call to a snake case function? (the bit I commented on above ☝️ )
If we do want snake casing (and I dont mind), then I'd rather do it here with these annotations if that will work?